Tests/AlgoliaSearch/ReindexCommandTest.php

```diff
@@ -83,7 +86,7 @@ public function reIndex($use_entity_alias, $safe = true)
 
         $input = new ArrayInput($options);
 
-        $this->command->run($input, new ConsoleOutput());
+        $this->command->run($input, new NullOutput());
```

Comment:
I understand, I wish we could get rid of the _"30 entities indexed"_ too. I tried on my local environnement, it didn't work so I expect problems with Travis as well.

Feel free to open a new pull request dedicated to that :)